svg/include/svg/element.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <sstream>
+
+namespace svg::geometry {
+
+    class Element {
```

Comment:
For me, there is two things mixed here. First, there is the definition of a geometrical object (rectangle, circle, ...) and second, there is the definition of an XML element. In my opinion, those two should be separated.